pkg/netceptor/tlsconfig.go

```diff
@@ -53,15 +59,22 @@ func (cfg tlsServerCfg) Prepare() error {
 	tlscfg.Certificates = []tls.Certificate{cert}
 
 	if cfg.ClientCAs != "" {
-		bytes, err := ioutil.ReadFile(cfg.ClientCAs)
+		caBytes, err := ioutil.ReadFile(cfg.ClientCAs)
 		if err != nil {
 			return fmt.Errorf("error reading client CAs file: %s", err)
 		}
 		clientCAs := x509.NewCertPool()
-		clientCAs.AppendCertsFromPEM(bytes)
+		clientCAs.AppendCertsFromPEM(caBytes)
 		tlscfg.ClientCAs = clientCAs
 	}
 
+	if len(cfg.PinnedClientCert) > 0 {
+		tlscfg.VerifyPeerCertificate, err = GetPeerCertificatePinVerifier(cfg.PinnedClientCert)
```

Comment:
for the quic connections, will this conflict with https://github.com/ansible/receptor/blob/2fb942e162a89be0a8793eb80691d0fa10249d08/pkg/netceptor/conn.go#L74

Reply:
You're right, this will indeed conflict.